### ISSUE TITLE
smart queue management

### ISSUE THREAD

**Issue by dtaht** [rchac/LibreQoS#4]
I do not have an issue with y'all calling this "libreqos". It's a well known term in this field. 

However, as it is actually a "smart queue management" system, also putting in a link to that might aid comprehension.

````
LibreQoS is a smart queue management (SQM) system[https://www.bufferbloat.net/projects/cerowrt/wiki/Smart_Queue_Management/] that allows ISPs to intelligently apply bandwidth rate limiting ...
````

PS (congrats on the ebpf support and cracking the 2gbps barrier!!!!)

**Comment by rchac** [comment]
I went ahead and updated the Readme to reflect that it is an SQM. Thank you for the suggestion. I would also be very open to renaming to LibreSQM given that might be more appropriate. And thank you! Very grateful to Jesper and xdp-cpumap-tc for making it possible.

**Comment by dtaht** [comment]
no, libreqos is a good name, a bridge to understanding what sqm is. I just gave you a plug over here:

https://forum.mikrotik.com/viewtopic.php?p=885037#p885037

The "discoverability" of bufferbloat, sqm, qos, are really miserable. I am hoping that tying the new word "responsiveness" to sqm perhaps we'll get somewhere.
